include/blmc_robots/real_finger.hpp

```diff
@@ -377,6 +378,11 @@ class NJointBlmcRobotDriver
         joint_modules_.set_position_control_gains(params.control_gain_kp,
                                                   params.control_gain_kd);
 
+        // For the calibration procedure we need to set the is_calibrated_ flag
+        // otherwise the robot cannot move during calibration.
+        // TODO: this is dirty, find a better solution.
+        is_calibrated_ = true;
```

Comment:
I was thinking about this option as well.  It would make the code a bit more complicated but I agree that it would be cleaner.  I am going to implement it like that.

Reply:
done